azure-pipelines.windows.yml

```diff
@@ -0,0 +1,178 @@
+name: Roaster Windows Build
+
+trigger:
+- master
+
+pool:
+  name: $(AGENT_POOL_NAME)
```

Comment:
Is this defined by you or by azp?
Default should be public resource, i.e. Microsoft hosted agent (even if there's no enough disk).

Reply:
I'm also think about use this in Github action, since the syntax is probably the same.

Reply:
It's a predefined pipeline variable. You should define it in your pipeline.

But azp supports overriding the agent pool. So having this variable here is not required.